proto/frequenz/api/microgrid/grid.proto

```diff
@@ -0,0 +1,22 @@
+// Contains definitions specific to grid connection points.
+//
+// Copyright:
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
+//
+// License:
+// MIT
+
+syntax = "proto3";
+
+package frequenz.api.microgrid.grid;
+
+// The grid connection point metadata.
+message Metadata {
+  // The rating of the fuse at the grid connection point, in amperes.
+  //
+  // Note that this limit applies to the magnitude of the current flowing both
+  // into or out of the grid connection point. E.g., a current _i_ A at the grid
+  // connection point has to follow the constraint:
+  // -max_grid_fuse_current <= i <= max_grid_fuse_current
+  uint32 max_grid_fuse_current = 1;
```

Comment:
Also minimal, but since this is already in the scope of the grid component category maybe we can just leave it at `max_fuse_current`.

Reply:
Done